spec/rmt/wizard_scc_page_spec.rb

```diff
@@ -21,6 +21,24 @@
 Yast.import 'Wizard'
 
 describe RMT::WizardSCCPage do
+  # Convenience method to inspect the tree of terms for the UI
+  def nested_id_term(id, term)
```

Comment:
would be nice to move those methods to a separate helper file. not a  must though.